tools/Common/IceRpc.Telemetry.Internal/Program.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc;
+using IceRpc.Retry;
+using IceRpc.Telemetry.Internal;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+const int timeout = 3000; // The timeout for the RPC call in milliseconds.
+const int maxAttempts = 3; // The maximum number of attempts to retry the RPC call.
+const string uri = "icerpc://localhost"; // The URI of the server.
+
+// Load the root CA certificate
+using var rootCA = new X509Certificate2("certs/cacert.der");
+var clientAuthenticationOptions = new SslClientAuthenticationOptions
+{
+    RemoteCertificateValidationCallback = (sender, certificate, chain, errors) =>
+    {
+        using var customChain = new X509Chain();
+        customChain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
+        customChain.ChainPolicy.DisableCertificateDownloads = true;
+        customChain.ChainPolicy.TrustMode = X509ChainTrustMode.CustomRootTrust;
+        customChain.ChainPolicy.CustomTrustStore.Add(rootCA);
+        return customChain.Build((X509Certificate2)certificate!);
+    }
+};
+
+try
+{
+    // Create a client connection
+    await using var connection = new ClientConnection(new Uri(uri), clientAuthenticationOptions);
+
+    // Create an invocation pipeline with the retry and deadline interceptors.
+    Pipeline pipeline = new Pipeline()
+        .UseRetry(new RetryOptions { MaxAttempts = maxAttempts })
+        .UseDeadline(defaultTimeout: TimeSpan.FromMilliseconds(timeout))
+        .Into(connection);
+
+    // Create a reporter proxy with this invocation pipeline.
+    var reporter = new ReporterProxy(pipeline);
+
+    // Upload the telemetry to the server.
+    await reporter.UploadAsync(new Telemetry(args));
+
+    // Shutdown the connection.
+    await connection.ShutdownAsync();
+}
+catch (Exception)
+{
+    // Consume all exceptions.
+}
```

Comment:
Even with the `protected override void LogEventsFromTextOutput` in the `TelemetryTask`, any exceptions will show up as warnings in the build console. As such, I just consume any exceptions. Is there a way to 'disable exceptions' via configuration like how you can enable or disable nullable?

Reply:
No, and you can't compare nullable with exceptions.